### ISSUE TITLE
Automation bypass button and gear menu option

### ISSUE THREAD

**Issue by Reaper10** [LMMS/lmms#2354]
In Automation set/clear record button #2083 added set/clear record button to the track I'd move then where mute/ solo buttons are now. Why? they aren't needed after all there isn't sound on this track. Where the set/clear record button are now there could be a Automation bypass button. This option should all so be in the gear menu. What does the Automation bypass button do? It just turns off the  Automation in the Automation track until the Automation bypass button is turned off. This way a user can ab the effect of a Automation track and this can be a part of live mode.

**Comment by tresf** [comment]
Thanks.  Please move this description to the first part of the bug report.

> What does the Automation bypass button do? It just turns off the Automation in the Automation track until the Automation bypass button is turned off. 

The rest is supplemental and should be reworded a bit as to not be a long run-on sentence.  Good effort here, let's keep improving these bug reports so that we aren't closing them on you.

-Tres

**Comment by Reaper10** [comment]
What does the Automation bypass button do? It just turns off the  Automation track until the bypass button is turned off.In Automation set/clear record button #2083 added set/clear record button to the track I'd move then where mute/ solo buttons are now, why? They aren't needed after all there isn't sound on this track. Where the set/clear record button are now there could be a Automation bypass button and this option should all so be in the gear menu.  This way a user can ab the effect of a Automation track and this can be a part of live mode.
 This any better?

**Comment by tresf** [comment]
No, fix the original post, don't pollute the thread with more unneeded text.  Delete that previous post please (and I'll delete my two posts).

Also, please separate ideas with line breaks.

You could type everything you ever wanted into a single run-on sentence and that would be ok if you didn't care who read it or how long it took for them to read it or if they even bother to read it like tldr, but it would greatly benefit the project, the tracker and your writing skills in general if you stop writing long runon sentences that really only get across a single point.  Get the point?
